### DISCUSSION TITLE
Cockpit does not light up at all.

### DISCUSSION THREAD

**Discussion by PapyCastor** [flybywiresim/aircraft#3968] [Help]
AFter 14.05 install and complete redraw of community folder, cockpit does not light up. Even though auto start Ctrl+e will start battery, engines, etc... nothing on the inbstruments.

**Comment by MapleNAV-TCASFAULT** [comment]
make sure to use the dev or exp version, and if it's still doing that with the dev or exp, you may have a mod conflict (remove all contents except for the a32nx folder)

**Reply by Paytheon** [reply]
Today I finally installed the new Stable version and it's working perfectly. Thanks to the dev team.

**Reply by PapyCastor** [reply]
Thank you for your replies, now working with the last stable version available March 16th  :)
